tests/rados/test_track_availibility_status.py

```diff
@@ -23,385 +25,647 @@
-        if not result_all_tests:
-            msg_error = f" The track availability tests are failed on {random_pool}"
-            log.error(msg_error)
-            return 1
-
-        msg_pool_deletion = f" The {random_pool} is selected to delete the pool "
-        log.info(msg_pool_deletion)
-        method_should_succeed(rados_object.delete_pool, random_pool)
-        new_pool_names.remove(random_pool)
-        msg_info = f"Remoed the {random_pool} from the pool list.The new pool list is-{new_pool_names}"
-        log.info(msg_info)
-        success, after_delete_pool_details = get_pool_availability_status(client)
-        if not success:
-            log.error("Failed to retrieve the pool available data")
-            return 1
-        if random_pool in after_delete_pool_details:
-            msg_error = f" The {random_pool} pool details exists in the availability-status output"
-            log.error(msg_error)
-            return 1
-        msg_info = (
-            f" The {random_pool} pool details not exists in the availability-status output."
-            f"The pool available status output is -{after_delete_pool_details}"
-        )
-        log.info(msg_info)
-        log.info("Deleting the created pools from the cluster")
-        for tmp_pool_name in new_pool_names:
-            method_should_succeed(rados_object.delete_pool, tmp_pool_name)
-        # Checking the log messages[TODO]
+            log.info("Case2: Retrieving the MON nodes")
+            mon_nodes = ceph_cluster.get_nodes(role="mon")
 
-        log.info(
-            "==== Case 4: Verification of the score value displayed on multiple pools  completed===="
-        )
-        # TODO
-        log.info(
-            "=== [IN-Progress] Case 5: Verification of track availability messages in the logs==="
-        )
+            init_time, _ = installer.exec_command(
+                cmd="sudo date '+%Y-%m-%dT%H:%M:%S.%3N+0000'"
+            )
+            init_time = init_time.strip()
+            msg_init_time = f"Case2: The initial time to check the logs is -{init_time}"
+            log.info(msg_init_time)
+            # Truncate the mon logs
+            rados_object.remove_log_file_content(mon_nodes, daemon_type="mon")
+            mon_obj.set_config(section="mon", name="debug_mon", value="20/20")
+            success, previous_pool_data = get_pool_availability_status(client)
+            if not success:
+                log.error("Case2:Failed to retrieve the pool available data")
+                return 1
+            previous_total_pool = len(previous_pool_data)
+            msg_pre_pool = f"Case2: The number of  pools before starting tests are - {previous_total_pool}"
+            log.info(msg_pre_pool)
+            new_pool_names = []
+            for pool_num in range(20):
+                rados_object.create_pool(pool_name=f"test_pool{pool_num}")
+                new_pool_names.append(f"test_pool{pool_num}")
```

Comment:
fill small amount of data in each pool created

Reply:
Filling the data after creating pool.